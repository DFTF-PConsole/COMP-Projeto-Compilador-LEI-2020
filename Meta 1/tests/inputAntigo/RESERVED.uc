keywords auto break case const continue default do enum extern float for goto inline long register restrict signed sizeof static struct switch typedef union unsigned volatile _Bool _Complex _Imaginary
types char int short double void
conditional else while if
return