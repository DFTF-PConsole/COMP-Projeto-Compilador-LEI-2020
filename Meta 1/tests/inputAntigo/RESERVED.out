ID(keywords)
RESERVED(auto)
RESERVED(break)
RESERVED(case)
RESERVED(const)
RESERVED(continue)
RESERVED(default)
RESERVED(do)
RESERVED(enum)
RESERVED(extern)
RESERVED(float)
RESERVED(for)
RESERVED(goto)
RESERVED(inline)
RESERVED(long)
RESERVED(register)
RESERVED(restrict)
RESERVED(signed)
RESERVED(sizeof)
RESERVED(static)
RESERVED(struct)
RESERVED(switch)
RESERVED(typedef)
RESERVED(union)
RESERVED(unsigned)
RESERVED(volatile)
RESERVED(_Bool)
RESERVED(_Complex)
RESERVED(_Imaginary)
ID(types)
CHAR
INT
SHORT
DOUBLE
VOID
ID(conditional)
ELSE
WHILE
IF
RETURN
